<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KopfrechnenStart</class>
 <widget class="QWidget" name="KopfrechnenStart">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>kopfrechnen</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QStackedWidget" name="stackedWidget_background">
       <property name="maximumSize">
        <size>
         <width>300</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QWidget{
  background-color: rgba(255,255,255,230);
 border-radius: 10px;
}

QComboBox{
 background-color: rgba(147,14,249);
 color: black;
border: 1px solid black;
}

QPushButton{
 border-radius: 5px;
border: 1px solid black;
background-color: rgba(147,14,249);
color: black;
}

QPushButton:hover{
	background-color: black;
color: white;
}

QCheckBox{
border-radius: 5px;
border: 1px solid black;
background-color: rgba(147,14,249);
color: black;
}
QTextEdit{
border-radius: 5px;
border: 1px solid black;
background-color: rgba(147,14,249);
color: black;
}

QPlainTextEdit{
border-radius: 5px;
border: 1px solid black;
background-color: rgba(147,14,249);
color: black;
}

</string>
       </property>
       <property name="currentIndex">
        <number>1</number>
       </property>
       <widget class="QWidget" name="stackedWidget_backgroundPage1">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QLabel" name="errorLabel">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QComboBox" name="comb_Level">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>50</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <item>
              <property name="text">
               <string>Level 1</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Level 2</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Level 3</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Level 4</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Level 5</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_3">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QCheckBox" name="cB_AddSub">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>Addition/Subtraktion</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="cB_MultDiv">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>Multiplikation/Division</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_4">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="bt_start">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>Start</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="bt_score">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>History</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="page">
        <widget class="QWidget" name="gridLayoutWidget">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>70</y>
           <width>261</width>
           <height>351</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="gridLayout_score"/>
        </widget>
        <widget class="QPushButton" name="btn_btM">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>440</y>
           <width>259</width>
           <height>15</height>
          </rect>
         </property>
         <property name="text">
          <string>Back to Menu</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="page_2">
        <property name="styleSheet">
         <string notr="true">QLineEdit{
border-radius: 5px;
border: 1px solid black;
background-color: rgba(147,14,249);
color: black;
}
</string>
        </property>
        <widget class="QTextEdit" name="te_time">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>40</y>
           <width>104</width>
           <height>70</height>
          </rect>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QTextEdit" name="te_score">
         <property name="geometry">
          <rect>
           <x>180</x>
           <y>40</y>
           <width>104</width>
           <height>70</height>
          </rect>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QTextEdit" name="te_task">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>210</y>
           <width>261</width>
           <height>70</height>
          </rect>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QLineEdit" name="te_solution">
         <property name="geometry">
          <rect>
           <x>70</x>
           <y>390</y>
           <width>141</width>
           <height>31</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="btn_solution">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>500</y>
           <width>80</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>Solve</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="page_3">
        <widget class="QPushButton" name="btn_back_end">
         <property name="geometry">
          <rect>
           <x>100</x>
           <y>450</y>
           <width>80</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>Back to Menu</string>
         </property>
        </widget>
        <widget class="QTextEdit" name="te_score_end">
         <property name="geometry">
          <rect>
           <x>60</x>
           <y>210</y>
           <width>161</width>
           <height>70</height>
          </rect>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
